documentation/modules/configuring/con-config-mirrormaker2-connectors.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// assembly-config-mirrormaker2.adoc
+
+[id='con-config-mirrormaker2-connectors-{context}']
+= Connector configuration
+
+[role="_abstract"]
+Use Mirrormaker 2.0 connector configuration for the internal connectors that orchestrate the synchronization of data between Kafka clusters.
+
+The following table describes connector properties and the connectors you configure to use them.
+
+.MirrorMaker 2.0 connector configuration properties
+[cols="4a,2,2,2",options="header"]
+|===
+
+|Property
+|sourceConnector
```

Comment:
Should we use the proper names: `MirrorSourceConnector, `MirrorCheckpointConnector` and `MirrorHeartbeatConnector`?

Reply:
They're too long so they don't present well. Also, I don't think it's as easy to read if they all start with Mirror. In the resource config, they're `sourceConnector` etc. Unless you have a strong opinion, I'm going to leave as is.

Reply:
Ok, that makes sense